lib/rules/media-feature-name-value-no-unknown/__tests__/index.js

```diff
@@ -0,0 +1,196 @@
+'use strict';
+
+const { messages, ruleName } = require('..');
+
+testRule({
+	ruleName,
+	config: [true],
+
+	accept: [
+		{
+			code: '@media screen and (min-width: 768px) {}',
+			description: 'Basic media feature',
+		},
+		{
+			code: '@media screen and (min-width: 0) {}',
+			description: 'Length of zero, without a unit',
+		},
+		{
+			code: '@media screen and ( min-resolution :  2dpcm ) {}',
+			description: 'A resolution',
+		},
+		{
+			code: '@media screen and ( -moz-min-resolution :  2dpcm ) {}',
+			description: 'A resolution with a vendor prefix',
+		},
+		{
+			code: '@media screen and (resolution: 10.1dpcm) {}',
+			description: 'A resolution with a floating point value',
+		},
+		{
+			code: '@media screen and (min-width: $sm) {}',
+			description: 'Non-standard syntax is not checked by this rule',
+		},
+		{
+			code: '@media (color) {}',
+			description: 'Boolean context',
+		},
+		{
+			code: '@media (color : 1) {}',
+			description: 'Integer value',
+		},
+		{
+			code: '@media (aspect-ratio : 1 / 1) {}',
+			description: 'Aspect ratio value',
+		},
+		{
+			code: '@media (aspect-ratio : 1.2) {}',
```

Comment:
related: #6575

Reply:
It is indeed possible to resolve that issue by adding a secondary rule option.
Something to add extra validation for "only integers separated by a `/` form a valid aspect ratio"

Although a dedicated rule would likely be less complex overal.
Adding it here opens the door for validation checks for all things that were invalid at some point in the past.